extensions-contrib/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureStorageDruidModule.java

```diff
@@ -118,4 +133,11 @@ public AzureStorage getAzureStorageContainer(
   {
     return new AzureStorage(cloudBlobClient);
   }
+
+  @Provides
+  @LazySingleton
+  public AzureCloudBlobDruidToCloudObjectLocationConverter getAzureCloudBlobToLocationConverter()
+  {
+    return new AzureCloudBlobDruidToCloudObjectLocationConverter();
+  }
```

Comment:
why not `bind(AzureCloudBlobDruidToCloudObjectLocationConverter.class).in(Scopes.LazySingleton)`